backend/compact-connect/tests/app/base.py

```diff
@@ -128,6 +129,70 @@ def _inspect_persistent_stack(
                 provider_users_user_pool_app_client['WriteAttributes'], ['email', 'family_name', 'given_name']
             )
             self._inspect_data_events_table(persistent_stack, persistent_stack_template)
+            self._inspect_ssn_table(persistent_stack, persistent_stack_template)
+
+    def _inspect_ssn_table(self, persistent_stack: PersistentStack, persistent_stack_template: Template):
+        ssn_key_logical_id = persistent_stack.get_logical_id(persistent_stack.ssn_table.key.node.default_child)
+        ingest_role_logical_id = persistent_stack.get_logical_id(
+            persistent_stack.ssn_table.ingest_role.node.default_child
+        )
+        api_query_role_logical_id = persistent_stack.get_logical_id(
+            persistent_stack.ssn_table.api_query_role.node.default_child
+        )
+        ssn_table_template = self.get_resource_properties_by_logical_id(
+            persistent_stack.get_logical_id(persistent_stack.ssn_table.node.default_child),
+            persistent_stack_template.find_resources(CfnTable.CFN_RESOURCE_TYPE_NAME),
+        )
+        ssn_key_template = self.get_resource_properties_by_logical_id(
+            ssn_key_logical_id, persistent_stack_template.find_resources(CfnKey.CFN_RESOURCE_TYPE_NAME)
+        )
+        # This naming convention is important for opting into future organization access logging
```

Comment:
Can you elaborate on this one? Not certain I understand what you mean by "future organization access logging."

Reply:
[This ticket](https://github.com/orgs/csg-org/projects/1/views/7?pane=issue&itemId=91125439&issue=csg-org%7CCompactConnect%7C397) will probably involve creating an [organization trail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/creating-trail-organization.html) in CloudTrail to create an audit log of data events on this table. We would do an organization trail rather than a cloud trail in the app environment account to enhance security - better to have these audit trails out of reach if an account compromise happens. One way to simplify cross-account dependencies is to use naming conventions, rather than explicit references to particular resources.

With a naming convention established, rather than having to maintain a list of resource identifiers across our test/prod accounts, and bringing them into configuration an the management account, we can just create an organization trail that logs data events for resources that follow a pattern like `ends with '-DataEventsLog'`, which will immediately start logging data events for resources like our SSN table without needing any complicated multi-account deployment strategies.

Reply:
Got it. That makes sense. Could we add a little more detail to this comment, if only to clarify that "organization access logging" means "AWS account event logging" (or similar)?